vocabs/oada/index.js

```diff
@@ -289,6 +289,21 @@ register('time', override('generic-data-value', {
   },
 }));
 
+register('led-status', override('generic-data-value', {
```

Comment:
This one isn't really a generic data value, it's just a boolean true/false.  Don't override generic-data-value here, just make it type: Boolean as a regular JSON schema.